File: codewars/c/get_planet_name_by_id.c
// https://www.codewars.com/kata/515e188a311df01cba000003/train/c

#include <stdio.h>

const char *getPlanetName(int id) {
	char *planets[] = {
		"Mercury", "Venus", "Earth", "Mars",
		"Jupiter", "Saturn", "Uranus", "Neptune"
	};

	return planets[id - 1];
}

int main(int argc, char *argv[]) {
	int id;
	scanf("%d", &id);

	printf("%s\n", getPlanetName(id));
}

File: codewars/go/get_planet_name_by_id.go
// https://www.codewars.com/kata/515e188a311df01cba000003/train/go

package main

import "fmt";

func GetPlanetName(ID int) string {
    planets := [8]string{
        "Mercury", "Venus", "Earth", "Mars",
        "Jupiter", "Saturn", "Uranus", "Neptune",
    };

    return planets[ID - 1];
}

func main() {
    var id = 0;
    fmt.Scanf("%d", &id);

    fmt.Println(GetPlanetName(id));
}
